docs/appendices/release-notes/unreleased.rst

```diff
@@ -61,6 +61,12 @@ Changes
   that concatenates array elements into a single string using a separator and
   an optional null-string.
 
+- Added :ref:`array_min <scalar-array-min>` scalar function
+  that finds minimal element in array.
+
+- Added :ref:`array_max <scalar-array-max>` scalar function
+  that finds maximal element in array.
+
```

Comment:
these two items could probably be collapsed into a single item

Reply:
Done

Reply:
@norosa @mfussenegger should I mention here, that only primitive types are supported? We have some types with implemented `compare`, for example `GeoPointType` and users might have an impression that they can find min/max for those types.

Reply:
Added note about primitive types in scalar-functions.rst